fixtures/dom/src/components/fixtures/text-inputs/index.js

```diff
@@ -1,62 +1,92 @@
 const React = window.React;
 
+import Fixture from '../../Fixture';
+import FixtureSet from '../../FixtureSet';
+import TestCase from '../../TestCase';
+import InputTestCase from './InputTestCase';
+
 class TextInputFixtures extends React.Component {
-  state = {
-    color: '#ffaaee',
-  };
+  render() {
+    return (
+      <FixtureSet
+        title="Inputs"
+        description="Common behavior across controled and uncontrolled inputs"
+      >
+        <TestCase title="Numbers in a controlled text field with no handler">
+          <TestCase.Steps>
+            <li>Move the cursor to after "2" in the text field</li>
+            <li>Type ".2" into the text input</li>
+          </TestCase.Steps>
 
-  renderControlled = (type) => {
-    let id = `controlled_${type}`;
+          <TestCase.ExpectedResult>
+            The text field's value should not update.
```

Comment:
Might change to ``{`The text field's value should not update.`}`` to make syntax highlighters happy.